upstairs/src/lib.rs

```diff
@@ -4669,7 +4679,7 @@ impl Downstairs {
             }
             // Now that we've collected jobs to retire, remove them from the map
             for &id in &retired {
-                self.ds_active.remove(&id).unwrap();
+                let _ = self.ds_active.remove(&id);
```

Comment:
Why eat the result, instead of panicking given the upstairs is in a state we don't expect?

Reply:
The unwrap moved into `remove`, so it will still panic, but we don't use the result.

Reply:
:+1: